microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -55,10 +67,55 @@ def check_dpdk_build(self, node: Node, log: Logger) -> None:
             f"TX-PPS ({tx_pps}) should have been greater than 2^20 (~1m) PPS."
         ).is_greater_than(2 ** 20)
 
-    def _execute_expect_zero(self, node: Node, cmd: str) -> str:
-        result = node.execute(cmd, sudo=True, shell=True)
-        result.assert_exit_code()
-        return result.stdout
+    @TestCaseMetadata(
+        description="""
+            Tests a basic sender/receiver setup.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_nic_count=2,
+            network_interface=Sriov,
+            min_count=2,
+        ),
+    )
+    def check_2node_snd_rcv(self, environment: Environment, log: Logger) -> None:
```

Comment:
1. use full name on method names. They will be test case name by default.
2. Start with "verify", so it can align with other test cases.